.eslintrc.js

```diff
@@ -2,7 +2,6 @@ module.exports = {
   parser: '@typescript-eslint/parser',
   extends: [
     'plugin:@typescript-eslint/recommended', // Uses the recommended rules from the @typescript-eslint/eslint-plugin
-    'prettier/@typescript-eslint', // Uses eslint-config-prettier to disable ESLint rules from @typescript-eslint/eslint-plugin that would conflict with prettier
```

Comment:
Isn't it good to have just Prettier caring about formatting and not eslint? I don't remember why I exactly added this line, but I'm pretty sure it had some reasons 😄 

Maybe it's outdated settings. I found just this https://robertcooper.me/post/using-eslint-and-prettier-in-a-typescript-project from April 25, 2020 also using the same settings

Reply:
I changed the settings because of updated dependencies, but we need the prettier/eslint integration because otherwise we're getting conflicts. We don't want eslint to do certain formatting that is then changed by prettier. This solves that problem.